sdk/tests/deqp/modules/shared/glsBuiltinPrecisionTests.js

```diff
@@ -5094,7 +5105,7 @@ var setParentClass = function(child, parent) {
      * @extends {glsBuiltinPrecisionTests.ArcTrigFunc}
      */
     glsBuiltinPrecisionTests.ATan = function() {
-        glsBuiltinPrecisionTests.ArcTrigFunc.call(this, 'atan', Math.atan, 5,
+        glsBuiltinPrecisionTests.ArcTrigFunc.call(this, 'atan', Math.atan, 4096.0,
```

Comment:
The native code relaxed the restriction of the precision by this code snippet: https://android.googlesource.com/platform/external/deqp/+/deqp-dev/modules/glshared/glsBuiltinPrecisionTests.cpp#2433